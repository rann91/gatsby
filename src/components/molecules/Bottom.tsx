/* @jsx jsx */
import { jsx, Box, Text, Link, Container } from 'theme-ui'

const Bottom = () => (
  <Box bg="white" pb={[7, null, null, 8]}>
    <Container>
      <Text
        as="p"
        px={5}
        sx={{ color: 'secondary', fontSize: 0, textAlign: 'center' }}>
        Made with{' '}
        <Link
          variant="footer"
          href="https://www.gatsbyjs.org"
          target="_blank"
          rel="noopener noreferrer">
          Gatsby
        </Link>{' '}
        and{' '}
        <Link
          variant="footer"
          href="https://strapi.io"
          target="_blank"
          rel="noopener noreferrer">
          Strapi
        </Link>
        . Design by{' '}
        <Link
          variant="footer"
          href="https://webflow.com"
          target="_blank"
          rel="noopener noreferrer">
          Webflow
        </Link>
        .
      </Text>
    </Container>
  </Box>
)

export default Bottom
