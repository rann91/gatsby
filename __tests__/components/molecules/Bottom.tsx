import { render } from '@testing-library/react'
import React from 'react'
import Bottom from '../../../src/components/molecules/Bottom'

describe('Bottom', () => {
  it('renders correctly', () => {
    const { container } = render(<Bottom />)

    expect(container).toMatchSnapshot()
  })
})
