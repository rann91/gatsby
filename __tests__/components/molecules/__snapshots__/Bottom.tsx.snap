// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bottom renders correctly 1`] = `
<div>
  <div
    class="css-1ca5803"
    data-testid="bottom"
  >
    <div
      class="css-1yrmf69"
    >
      <p
        class="css-10im984-Bottom"
      >
        Made with
         
        <a
          class="css-vurnku"
          href="https://www.gatsbyjs.org"
          rel="noopener noreferrer"
          target="_blank"
        >
          Gatsby
        </a>
         
        and
         
        <a
          class="css-vurnku"
          href="https://strapi.io"
          rel="noopener noreferrer"
          target="_blank"
        >
          Strapi
        </a>
        . Design by
         
        <a
          class="css-vurnku"
          href="https://webflow.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          Webflow
        </a>
        .
      </p>
    </div>
  </div>
</div>
`;
